NOTICE

```diff
@@ -1,5 +1,5 @@
 Code.org tutorial platform
-Copyright 2013 Code.org
+Copyright 2016 Code.org
```

Comment:
Not a stupid question at all.
At least, I don't know (nor does a quick search reveal) how to do so in a markdown file.
Ignore the suggestion for now.